### DISCUSSION TITLE
Privacy Sanbox display wrong SameSite attribute value

### DISCUSSION THREAD

**Discussion by betagoo** [GoogleChromeLabs/ps-analysis-tool#613] [Support Forum] [answered]
Privacy Sanbox display wrong SameSite attribute value.
Into Application tab, the cc_cookie has SameSite value set to Lax.
This is correct.
Lax was set when cookie was created.

![Snipaste_2024-02-21_11-12-40](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/3964399/fb8a0407-a9ae-459d-8575-38f194d456e5)

Into Privacy Sanbox tab, the cc_cookie has SameSite attribute empty.
Is this the bug?
Why in this view is not displaying value Lax?

![Snipaste_2024-02-21_11-14-35](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/3964399/b6c701ec-9fcd-4128-96dc-7fe10bbaeeb5)

**Comment by milindmore22** [comment, marked as answer]
Hello @betagoo 

Thank you for reporting this issue, our team is looking into the same.
In the meantime, to get more accurate & additional details about the site being analyzed, please enable the CDP feature from the PSAT extension settings.

You can enable CDP in two ways from [PSAT settings](https://github.com/GoogleChromeLabs/ps-analysis-tool/wiki/PSAT-Settings-and-Permissions#enabling-chrome-devtools-protocol-in-psat) and [PSAT popup](https://github.com/GoogleChromeLabs/ps-analysis-tool/wiki/PSAT-Extension-Popup). Once you enable the CDP, you will notice a “reload” button in the popup or in the extension settings, use it to start a debugging session of your chrome instance. (Please note that the “reload” button will reload all your Chrome tabs)

Once you enable CDP, the “Privacy Sandbox” tab should reflect the correct “Lax” value for the “cc_cookie”, as shown in the screenshot below

![Screen Shot 2024-02-23 at 6 49 20 PM](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/1132541/7c14ff69-c110-46ee-af78-5806deda4bcb)


If you encounter any further issues or have suggestions for improvement, please don't hesitate to start a [discussion](https://github.com/GoogleChromeLabs/ps-analysis-tool/discussions/categories/support-forum).

**Comment by milindmore22** [comment]
@betagoo I will be converting this issue to support forum discussion, For future reference, please [create support discussion](https://github.com/GoogleChromeLabs/ps-analysis-tool/discussions/new?category=support-forum) if you have any question about the PSAT.
